scripts/globals/status.lua

```diff
@@ -794,7 +794,9 @@ dsp.effect =
     CHAINBOUND               = 798,
     SKILLCHAIN               = 799,
     DYNAMIS                  = 800,
-    MEDITATE                 = 801, -- Dummy effect for SAM Meditate JA
+    MEDITATE                 = 801,
+    DAKEN                    = 909,
```

Comment:
wrong table. goes in dsp.mods. Also update the SPARE comments here.

Also looks like you moved the comment that is meant to be next to the MEDITATE effect.

Reply:
My bad, thanks!